core/src/test/java/io/paradoxical/cassieq/unittests/TestQueueContext.java

```diff
@@ -28,41 +28,65 @@
 public class TestQueueContext {
 
     @NonNull
-    private final QueueName queueName;
+    @Getter
+    private QueueName queueName;
 
     @NonNull
     @Getter
-    private final QueueDefinition queueDefinition;
+    private QueueDefinition queueDefinition;
 
     @NonNull
-    private final Reader reader;
-    private final AccountName accountName;
-    private final Injector injector;
+    @Getter
+    private Reader reader;
+
+    @Getter
+    private AccountName accountName;
+
+    @Getter
+    private Injector injector;
 
     @Getter
     private QueueDataContext context;
 
     @Getter
-    private final QueueRepository queueRepository;
+    private QueueRepository queueRepository;
```

Comment:
I'll give it a shot, but the reason has to do with the expectation that one constructor creates a queue, and the other doesn't. to split up that logic i needed helper methods, and the final doesn't work unless its explicity done in the constructor (even though its effectively final anyways)